grape-swagger-jsonapi-resources.gemspec

```diff
@@ -36,4 +36,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "rubocop"
+  spec.add_development_dependency "kaminari"
```

Comment:
Do we want to lock ourselves into kaminari? It might be nice to investigate this: https://github.com/ddnexus/pagy for example.

Reply:
Not at all. But this is the one that came to hand initially and which I'm using in the stock orders project. Easy to make an adaptor later if we want. It's only used in the specs as the code itself just does duck typing.